peps/pep-0772.rst

```diff
@@ -189,13 +188,14 @@ Election of the council
 
 A council election consists of two phases:
 
-* Phase 1: A member of the Packaging community (defined later in this document)
-  can self-nominate themselves for the council elections.
-* Phase 2: Each Packaging community member can assign zero to five stars to
-  each candidate. Voting is performed anonymously. The outcome of the vote is
-  determined using the `STAR voting system`_, modified to use the `Multi-winner
-  Bloc STAR`_ approach. If a tie occurs, it may be resolved by mutual agreement
-  among the candidates, or else the winner will be chosen at random.
+* Phase 1: A voting member (defined later in this document) can self-nominate
+  themselves for the council elections. Such a nomination must include
+  information about the member's relevant affiliations.
```

Comment:
We don't say anything about non-self-nominations, but maybe we should allow voting members to also nominate other people, whether those nominees are themselves voting members or not (similar to how the SC and core dev nominations work).

Reply:
I'll add this to a follow up PR